pkg/apb/types.go

```diff
@@ -224,6 +224,8 @@ const (
 	ClusterKey = "cluster"
 	// NamespaceKey parameter name passed to APBs
 	NamespaceKey = "namespace"
+	// UsernameKey parameter name passed to APBs
+	UsernameKey = "username"
```

Comment:
@eriknelson @shawn-hurley is this supposed to be `_apb_username` or is `username` okay?

Reply:
Would it make more sense place the username [here](https://github.com/openshift/ansible-service-broker/blob/master/pkg/broker/types.go#L33) and set it with the others [here](https://github.com/openshift/ansible-service-broker/blob/master/pkg/broker/broker.go#L592) and NOT put it in the ServiceInstance?  I put it in the serviceInstance so that it retains the username info in the instance... but I can change it.

Reply:
@johnkim76 I think it's okay on the ServiceInstance, Since an instance should never have multiple users requesting it.

Reply:
Consider the scenario where the broker is set to `auto_esclate` UserA and UserB have enough access in the target namespace. UserA creates a service `postgresql` and then UserB creates a binding for that service. That would be two users requesting the same service instance.

I agree @johnkim76 that this key should be _apb_requesting_user  and should behave similarly those that you posted above.